README.md

```diff
@@ -85,24 +68,28 @@ catch( Exception e )
     Debug.LogError( e );
 }
 ```
+In the above snippet, if an account does not exist a Wallet and key pair will be created.
 
-Calling `GetAccount` with the existing account index, will retrieve the account stored on the device.
+Calling `GetAccount` with the existing account index will retrieve the account stored on the device.
 
 ```csharp
 if( kinClient.HasAccount() )
-    account = kinClient.GetAccount( 0 );
+    account = kinClient.getAccount( 0 );
 ```
 
-You can delete your account from the device using `DeleteAccount`, but beware! you will lose all your existing KIN if you do this.
+**Warning:** You can delete an account from the device using `deleteAccount`, but beware! The device will lose the account's private key and subsequently will lose access to the Kin stored in the account on the Kin Blockchain.
 
 ```csharp
 kinClient.DeleteAccount( int index );
 ```
 
 ## Onboarding
 
-Before an account can be used on the configured network, it must be created on the blockchain and funded with the native network asset. This step must be performed by a service, see the [Kin SDK documentation](https://github.com/kinecosystem/kin-sdk-android/blob/master/README.md) for details. When working with the Kin test servers, you can use the friendbot service (see the `KinOnboarding` class in the Unity demo project) to get your test accounts setup and funded.
+At this point in the process your Unity client has created an `account` but the Kin Blockchain does not yet know about it. In the production environment Kin clients are not authorized to create new accounts on the Kin Blockchain. Your server will be authorized to create accounts on their behalf using the [Kin SDK for Python](https://kinecosystem.github.io/kin-website-docs/docs/documentation/python-sdk).  
```

Comment:
> In the production environment Kin clients are not authorized to create new accounts
Clients cant do it on testnet as well.

And its not the clients are not authorized, the functionality is just not exposed on the client sdk